sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -75,6 +76,19 @@ private[sql] object JDBCRelation {
     }
     ans.toArray
   }
+
+  def getEffectiveProperties(
```

Comment:
@tribbloid I think Holden's comment still stands -- see how other methods wrap args. I also don't think you need to fully-qualify `scala.collection.Map` here?